pkg/execute/executor.go

```diff
@@ -118,8 +118,9 @@ func (e *DefaultExecutor) Execute(ctx context.Context) interactive.CoreMessage {
 		return empty // user specified different target cluster
 	}
 
-	// commands below are executed only if the channel is authorized
-	if !e.conversation.IsAuthenticated {
+	// commands below are executed only if the channel is configured
+	if !e.conversation.IsKnown {
+		e.log.Infof("Unknown conversation with ID %q. Returning empty message...", e.conversation.ID)
```

Comment:
actually, ID will be usually empty, so maybe something like:
```suggestion
		e.log.Info("Unknown conversation. Returning empty message...")
```